tests/common/common.h

```diff
@@ -753,6 +753,56 @@ inline bool kernel_supports_wg_size(sycl_cts::util::logger& log,
     }                                                                    \
   }
 
+/** \brief Tests the result of using unary operator op with operand val,
+ *         while storing the results in res. */
+#define UNARY_INDEX_KERNEL_TEST(op, val, res)                   \
+  do {                                                          \
+    val = op res;                                               \
```

Comment:
This is an outdated version of the code, Ronan earlier left the same comment (https://github.com/KhronosGroup/SYCL-CTS/pull/478#discussion_r1083868486) and I have since fixed the mistake.

Reply:
Sorry, I should have removed this comment. I started reviewing before you committed the fix but didn't finish and GitHub posted my old comment with this one - https://github.com/KhronosGroup/SYCL-CTS/pull/478#discussion_r1088207188.